apps/deploy-web/src/queries/useAnonymousUserQuery.ts

```diff
@@ -21,9 +21,14 @@ export function useAnonymousUserQuery(id?: string, options?: { enabled?: boolean
   const [userState, setUserState] = useState<{ user?: UserOutput; isLoading: boolean; token?: string }>({ isLoading: !!options?.enabled });
 
   useWhen(options?.enabled && !userState.user, async () => {
-    const { data: fetched, ...rest } = await userHttpService.getOrCreateAnonymousUser(id);
-    const token = "token" in rest ? rest.token : undefined;
-    setUserState({ user: fetched, token, isLoading: false });
+    try {
+      const { data: fetched, ...rest } = await userHttpService.getOrCreateAnonymousUser(id);
+      const token = "token" in rest ? rest.token : undefined;
+      setUserState({ user: fetched, token, isLoading: false });
+    } catch (error) {
+      console.error(error);
```

Comment:
I kinda missed how this log ended up here. However just want to point out that it's more/less useless. If there's an error it should be added to the state above and handled somewhere in ui.

Reply:
This is to handle the case where the id of the anonymous user is in the browser but not in db anymore, like staging where we clear the users from time to time and it just makes the ui stuck in loading screen. I'm not sure what good it would do to show the user that this didn't work as it's supposed to be a hidden logic. It is supposed to create one if it doesn't find it so I'm not sure why it throws.

Reply:
I can change it to `Sentry.captureException(error);`